### ISSUE TITLE
Complete sensitive scope app verification

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#4608]
From Google:

> Hello Google Developer,
>
> You are receiving this email because we’ve detected that your project(s) currently request sensitive user data via OAuth scopes which haven’t yet been verified as required by the OAuth 2.0 policy.
>
> If your app is not verified by September 30, 2021, your users will receive an unverified app screen message and your app will be subject to a 100 user cap for new users for access to sensitive or restricted data.

This is for the Google Drive integration. I did the verification recently for the Google Data Studio connector (https://github.com/UDub-Impact/OData-Connector/issues/6) so I can take it on.

**Comment by lognaturel** [comment]
I believe that Collect doesn't actually use this [anymore?]. The Google Drive integration uses accounts registered on the device so the user just has to authorize Google to add any new account. As far as I can tell, the user never has to explicitly give Collect permissions to write to Drive/Sheets. Feels a little weird. Does that sound right, @grzesiek2010?

The consent screen would look like [this](https://stackoverflow.com/questions/60954001/how-can-i-configure-google-oauth-consent-screen-to-not-show-checkboxes) with something like "ODK wants to access your Google Account." I can't find a way for Collect to show anything like that.

Frustratingly, I can't find any way from the Google Cloud Console to get any hints about how this screen is used. What I have found is how many tokens are granted. It looks like ~2 a day. I'm not sure what that means since I don't know at what frequency they expire but that also doesn't seem like the kind of usage we'd expect with Collect's Drive integration.

I'm thinking now that it might just be for old versions of Aggregate or something like that. There used to be a way to use a Google account to log in and that was deprecated a while ago. The side effect of not getting verified is that token granting gets throttled. Given so few are being granted anyway, it seems it wouldn't matter. I'll close for now.

**Comment by grzesiek2010** [comment]
> As far as I can tell, the user never has to explicitly give Collect permissions to write to Drive/Sheets.

you are asked to grant that permission if you use Google Drive for the first time. This issue contains a screenshot: https://github.com/getodk/collect/issues/4701 isn't it the same?

**Comment by lognaturel** [comment]
Sigh. Thanks, @grzesiek2010. I thought I’d cleared the permissions but I guess I didn’t do it right. So maybe the token grant is one-time per account in which case 2-ish per day makes sense. Ok, will keep working on this.

**Comment by lognaturel** [comment]
The application is complete and submitted.

**Comment by lognaturel** [comment]
I put in the application just for the Drive scope used by Collect. 

We got an automated response saying the project currently has access to the following scopes I did not include in the application:

- https://www.googleapis.com/auth/photos
- https://www.googleapis.com/auth/spreadsheets 

The first I find no record of. I think it may have been for a Fusion Tables or Google Earth publish feature in Aggregate that was removed.

The second is for Aggregate publish to Google Sheets. The OAuth screen isn’t shown for that because it uses a service worker. It may or may not still require this approval. I’ve discussed with @yanokwa and we’re moving forward with just the Drive scope for now.

**Comment by yanokwa** [comment]
I looked at the credential metrics and we haven't see any sheet usage from a non ODK-Collect OAuth client in the last 30 days. so this feels like low risk. Also Aggregate is end-of-life. Also if there is pushback we can re-add this scope.

**Comment by lognaturel** [comment]
Thanks again @grzesiek2010 for helping me identify why this was needed.

We've been approved. Publish to Spreadsheet from Aggregate still seems to work so I don't know what that second scope was used for. Now I vaguely remember that we may at some point have used both the Drive and Spreadsheet APIs in Collect.

I've verified the OAuth screen has been updated:
![Screenshot_1631126537](https://user-images.githubusercontent.com/967540/132566765-c04748e5-5204-4f4f-8202-c11d22e43fc3.png)
